packages/lesswrong/components/common/SubscribedUsersFeed.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Components, registerComponent } from "../../lib/vulcan-lib/components";
+
+const styles = (theme: ThemeType): JssStyles => ({
+})
+
+const SubscribedUsersFeed = ({classes}: {
+  classes: ClassesType<typeof styles>,
+}) => {
+  const { MixedTypeFeed, SuggestedFeedSubscriptions } = Components;
+  /*return <ResolverPostsList
+    resolverName="PostsBySubscribedAuthors"
+    limit={13}
+    fallbackText="Visits users' profile pages to subscribe to their posts and comments."
+    showLoadMore
+  />*/
+  
+  return <div>
+    <SuggestedFeedSubscriptions />
+    <MixedTypeFeed
+      resolverName={"SubscribedFeed"}
+      firstPageSize={10}
+      pageSize={20}
+      sortKeyType="Date"
+      renderers={{
+        postCommented: {
+          fragmentName: "SubscribedPostAndCommentsFeed",
+          render: (postCommented: SubscribedPostAndCommentsFeed) => {
+            return <Components.RecentDiscussionThread
+              key={postCommented.post._id}
+              post={postCommented.post}
+              comments={postCommented.comments}
+              refetch={()=>{} /*TODO*/}
```

Comment:
Is there something that needs to be done here?

Reply:
Maybe, Jim wrote this bit. I think it's fine for the current state of things